Xamarin.Essentials/Battery/Battery.android.cs

```diff
@@ -38,7 +27,7 @@ public static double ChargeLevel
         {
             get
             {
-                ValidateBatteryStatsPermission();
+                Permissions.EnsureDeclared(PermissionType.Battery);
```

Comment:
We don't need to cache this as it is quick, and if we really need to, we can do that in the permissions api.